mesheryctl/internal/cli/root/system/token.go

```diff
@@ -0,0 +1,170 @@
+package system
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var ctx string
+var tokenCmd = &cobra.Command{
+	Use:   "token",
+	Short: "Manage Meshery user tokens",
+	Long: `
+	Manipulate user tokens and their context assignments in your meshconfig`,
+	Args: cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
+			return errors.New(utils.SystemError(fmt.Sprintf("invalid command: \"%s\"", args[0])))
+		}
+		return nil
+	},
+}
+
+var addTokenCmd = &cobra.Command{
+	Use:   "add",
```

Comment:
@navendu-pottekkat we’re using add here and not create like we do for context?

Reply:
Yes, create is something that eventually means creation, which is not exactly the case with tokens, we aren't creating a token, we are adding it to the config. That is why @navendu-pottekkat replaced to with "add" instead of "create"